Elasticity.C

```diff
@@ -1035,12 +1035,6 @@ bool ElasticityNorm::finalizeElement (LocalIntegral& elmInt)
 }
 
 
-void ElasticityNorm::addBoundaryTerms (Vectors& gNorm, double energy) const
```

Comment:
do we actually override this anywhere? i don't think so, thus we could consider devirtualizing the method.